addons/OPT/GPS/fn_clientInit.sqf

```diff
@@ -1,120 +1,60 @@
 #include "macros.hpp"
-[] call FUNC(initCBASettings);
 
-// Inspired by AAW's UnitTracker
-
-GVAR(playerCounter) = 0;
-GVAR(currentIcons) = [];
-GVAR(blockUpdate) = false;
-GVAR(currentHoverGroup) = grpNull;
-GVAR(currentHoverVehicle) = objNull;
-GVAR(groupInfoPFH) = -1;
-GVAR(vehicleInfoPFH) = -1;
-GVAR(lastFrameTriggered) = diag_frameNo;
-
-GVAR(processedIcons) = [];
-GVAR(lastProcessedIcons) = [];
-
-
-// Register Beam dialog map to show unit markers too. Only try doing this once the mission actually started, so everything is hopefully in place.
-// Because we can only do this for an existing display/control, we gotta wait until it actually exists
 ["missionStarted", {
-    [{
-        ((findDisplay 444001) displayCtrl 10007) call CFUNC(registerMapControl);
-    }, {!(isNull ((findDisplay 12) displayCtrl 51))}] call CFUNC(waitUntil);
-
-}] call CFUNC(addEventhandler);
-
-
-
-
-
-/*
-The purpose of this stateMachine is to draw mapicons for each unit, group and vehicle whenever a map control is open.
-It draws a single icon per frame to prevent it from hogging resources, while still guaranteeing maximum drawrate.
-Once all icons have been drawn, it cycles back through them in perpetuity. To do this, it has two steps:
-Step 1 initializes _units and _vehicles with what needs to be drawn (init), step 2 draws a single icon (addIcons).
-*/
-GVAR(iconStatemachine) = call CFUNC(createStatemachine);
-
-[GVAR(iconStatemachine), "init", {
-    private _units = allUnits select {[_x] call FUNC(isUnitVisible)};
-
-    // TODO: For now, we ignore empty vehicles
-    // private _vehicles = vehicles select {[_x] call FUNC(isVehicleVisible)};
-    private _vehicles = [];
-
-    // Figure out what icons we had last time. Maybe it's different this time around and we need to remove some icons!
-    GVAR(lastProcessedIcons) = (CGVAR(MapGraphics_MapGraphicsGroup) call CFUNC(allVariables)) select {(_x find toLower QGVAR(IconId)) == 0};
-
-    {
-        DUMP("ICON REMOVED: " + _x);
-        [_x, "hoverin"] call CFUNC(removeMapGraphicsEventHandler);
-        [_x, "hoverout"] call CFUNC(removeMapGraphicsEventHandler);
-        [_x] call CFUNC(removeMapGraphicsGroup);
-    } count (GVAR(lastProcessedIcons) - GVAR(processedIcons));
-
-    // Now we got a clean slate again, let's start the cycle.
-    GVAR(processedIcons) = [];
-
-    if (_units isEqualTo [] && _vehicles isEqualTo []) then {
-        "init"
-    } else {
-        ["addIcons", [_units, _vehicles]]
+    // Create markers for all players already here in case of JIP
+    allPlayers apply {
+        DUMP(_x);
```

Comment:
Würde hier noch ein `DUMP` drüber setzen, damit man im Log später besser durch blickt:
```
DUMP("Adding initial unit...");
```
oder sowas in der Art